spring-kafka/src/main/java/org/springframework/kafka/listener/ConsumerProperties.java

```diff
@@ -116,6 +117,12 @@ public class ConsumerProperties {
 
 	private boolean checkDeserExWhenValueNull;
 
+	/**
+	 * Container group protocol that will be set on the Kafka Consumer.
+	 * Default to classic consumer protocol.
+	 */
+	private String groupProtocol;
```

Comment:
Isn't there some `enum` to rely on?
Or we might introduce one as an inner in this `ConsumerProperties` class...